docs/06-actors-actions.md

```diff
@@ -0,0 +1,453 @@
+  }
+}
+```
+
+### Core update
+
+[[Actor.update]] is the core update logic to prepare the frame. **This is an advanced method override.** You can take over the whole update loop by overriding this method. Use `super.update(engine, delta)` to invoke the core logic or leave it out to completely customize the update logic.
+
+```ts
+/**
+ * DANGER: This is for advanced users to totally override update logic.
+ */
+public update(ctx: CanvasRenderingContext2D, delta: number) {
+
+  // Invoke the core logic
+  // Or leave it out to completely override
+  super.update(engine, delta);
+
+  // Perform custom update logic
+  // ...
+}
+```
+
+### Post-update
+
+[[Actor.onPostUpdate]] is called after [[Actor.update]] to prepare state for the _next_ frame. Things that need to be updated include state, drawing, or position.
+
+<docs-note>This is the recommended method to override for adding update logic to your actors since it runs after Excalibur has done all the update logic for the frame and before things get drawn to the screen.</docs-note>
+
+```ts
+class Player extends Actor {
+  /**
+   * RECOMMENDED: Runs after "core" update logic, before the next frame.
+   * Usually this is what you want!
+   */
+  public onPostUpdate(engine: ex.Engine, delta: number) {
+    // check if player died
+    if (this.health <= 0) {
+      this.kill()
+      return
+    }
+  }
+}
+```
+
+## Drawing actors
+
+Actors by default have no associated [drawings](/docs/drawings), meaning that they will be rendered without any graphics unless you've assigned a default [[Actor.color]] or attached a drawing. If an actor has a color set, it will draw a box in that color. This is useful only at the beginning of development when you're just tinkering but for most games you'll need to add sprites, animations, and other drawings.
+
+### Working with textures & sprites
+
+Think of a [[Texture|texture]] as the raw image file that will be loaded into Excalibur. In order for it to be drawn
+it must be converted to a [Sprite](/docs/drawings#sprites).
+
+A common usage is to load a [[Texture]] and convert it to a sprite for an actor. If you are using the [asset loader](http://localhost:8000/docs/assets) to
+pre-load assets, you can use [[Actor.addDrawing]] to add a texture directly, which will internally convert it to a sprite on your behalf. You can also create a
+[[Texture.asSprite|sprite from a Texture]] to create a sprite instance to pass to `addDrawing`.
+
+```ts
+// assume Resources.TxPlayer is a 80x80 png image
+
+public onInitialize(engine: ex.Engine) {
+
+   // set as the "default" drawing
+   this.addDrawing(Resources.TxPlayer);
+
+   // you can also set a Sprite instance to draw
+   this.addDrawing(Resources.TxPlayer.asSprite());
+}
+```
+
+### Working with animations
+
+A [sprite sheet](/docs/drawings#sprite-sheets) holds a collection of sprites from a single [[Texture]].
+Use [[SpriteSheet.getAnimationForAll]] to quickly generate an [Animation](/docs/drawings#animations).
```

Comment:
This could use a little more explanation, mostly about whether adding the Animation passed to `addDrawing` will automatically make Excalibur animate the Actor.